### PR TITLE
Filaments picking

### PR DESCRIPTION
**Pull request by fonsecareyna82**
- Adding filaments picking options
- Adding a manual picking protocol using napari-boxmanager

### CODE REVIEW

File: sphire/tests/test_protocol_cryolo.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def setUpClass(cls):
         setupTestOutput(cls)
 
     def testConvertCoords(self):
+        import time
```
Comment (pconesa) [inline sphire/tests/test_protocol_cryolo.py:56]: What is this?

**Review by pconesa — APPROVED** [review]

**Comment by azazellochg** [comment]
I can see a lot of code duplication and base class is not used

**Review by pconesa — APPROVED** [review]

**Comment by pconesa** [comment]
Sorry, just realized Grigori's mesage... @fonsecareyna82 could you please have a look and reduce code duplication?

**Comment by azazellochg** [comment]
No problem, I can help and make another PR later.

**Pull request merged**